{"name": "Test Empty Payer Name Validation", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.TargetClosedError: Locator.fill: Target page, context or browser has been closed\nCall log:\n  - waiting for locator(\"input[name=\\\"amt\\\"]\")", "trace": "self = <testCases.errorMessageValidation_test.TestClientValidation object at 0x000001D6FE11A5D0>\npage_with_screenshot = <pageObjects.Client_test.ClientTest object at 0x000001D6FE2138C0>\n\n    @allure.story(\"Payer Name Validations\")\n    @allure.severity(allure.severity_level.CRITICAL)\n    @allure.title(\"Test Empty Payer Name Validation\")\n    @pytest.mark.ui\n    @pytest.mark.critical\n    def test_empty_payer_name_validation(self, page_with_screenshot: ClientTest):\n        \"\"\"Test Payer Name Field Validation\"\"\"\n        try:\n            with allure.step(\"Filling Client Code\"):\n                page_with_screenshot.fill_client_code(\"LPSD1\")\n    \n            with allure.step(\"Selecting Environment\"):\n                page_with_screenshot.select_environment(\"STAGING\")\n    \n            with allure.step(\"Clicking Show Detail\"):\n                page_with_screenshot.click_show_detail()\n    \n    \n    \n            with allure.step(\"Entering Payment Details\"):\n>               page_with_screenshot.fill_amount(\"10\")\n\ntestCases\\errorMessageValidation_test.py:33: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npageObjects\\Client_test.py:49: in fill_amount\n    self.amount_input.fill(amount)\n.venv\\Lib\\site-packages\\playwright\\sync_api\\_generated.py:15808: in fill\n    self._sync(\n.venv\\Lib\\site-packages\\playwright\\_impl\\_locator.py:208: in fill\n    return await self._frame.fill(self._selector, strict=True, **params)\n.venv\\Lib\\site-packages\\playwright\\_impl\\_frame.py:527: in fill\n    await self._channel.send(\"fill\", locals_to_params(locals()))\n.venv\\Lib\\site-packages\\playwright\\_impl\\_connection.py:61: in send\n    return await self._connection.wrap_api_call(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <playwright._impl._connection.Connection object at 0x000001D6FE12FA10>, cb = <function Channel.send.<locals>.<lambda> at 0x000001D6FE43F560>, is_internal = False\n\n    async def wrap_api_call(\n        self, cb: Callable[[], Any], is_internal: bool = False\n    ) -> Any:\n        if self._api_zone.get():\n            return await cb()\n        task = asyncio.current_task(self._loop)\n        st: List[inspect.FrameInfo] = getattr(task, \"__pw_stack__\", inspect.stack())\n        parsed_st = _extract_stack_trace_information_from_stack(st, is_internal)\n        self._api_zone.set(parsed_st)\n        try:\n            return await cb()\n        except Exception as error:\n>           raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE           playwright._impl._errors.TargetClosedError: Locator.fill: Target page, context or browser has been closed\nE           Call log:\nE             - waiting for locator(\"input[name=\\\"amt\\\"]\")\n\n.venv\\Lib\\site-packages\\playwright\\_impl\\_connection.py:528: TargetClosedError"}, "description": "Test Payer Name Field Validation", "steps": [{"name": "Filling Client Code", "status": "passed", "start": 1742284025417, "stop": 1742284025770}, {"name": "Selecting Environment", "status": "passed", "start": 1742284025770, "stop": 1742284026097}, {"name": "Clicking Show Detail", "status": "passed", "start": 1742284026097, "stop": 1742284026497}, {"name": "Entering Payment Details", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.TargetClosedError: Locator.fill: Target page, context or browser has been closed\nCall log:\n  - waiting for locator(\"input[name=\\\"amt\\\"]\")\n\n", "trace": "  File \"C:\\Users\\Bhargava\\PycharmProjects\\newpg\\testCases\\errorMessageValidation_test.py\", line 33, in test_empty_payer_name_validation\n    page_with_screenshot.fill_amount(\"10\")\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^\n  File \"C:\\Users\\Bhargava\\PycharmProjects\\newpg\\pageObjects\\Client_test.py\", line 49, in fill_amount\n    self.amount_input.fill(amount)\n    ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"C:\\Users\\Bhargava\\PycharmProjects\\newpg\\.venv\\Lib\\site-packages\\playwright\\sync_api\\_generated.py\", line 15808, in fill\n    self._sync(\n    ~~~~~~~~~~^\n        self._impl_obj.fill(\n        ^^^^^^^^^^^^^^^^^^^^\n            value=value, timeout=timeout, noWaitAfter=no_wait_after, force=force\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n        )\n        ^\n    )\n    ^\n  File \"C:\\Users\\Bhargava\\PycharmProjects\\newpg\\.venv\\Lib\\site-packages\\playwright\\_impl\\_sync_base.py\", line 115, in _sync\n    return task.result()\n           ~~~~~~~~~~~^^\n  File \"C:\\Users\\Bhargava\\PycharmProjects\\newpg\\.venv\\Lib\\site-packages\\playwright\\_impl\\_locator.py\", line 208, in fill\n    return await self._frame.fill(self._selector, strict=True, **params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Bhargava\\PycharmProjects\\newpg\\.venv\\Lib\\site-packages\\playwright\\_impl\\_frame.py\", line 527, in fill\n    await self._channel.send(\"fill\", locals_to_params(locals()))\n  File \"C:\\Users\\Bhargava\\PycharmProjects\\newpg\\.venv\\Lib\\site-packages\\playwright\\_impl\\_connection.py\", line 61, in send\n    return await self._connection.wrap_api_call(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    ...<2 lines>...\n    )\n    ^\n  File \"C:\\Users\\Bhargava\\PycharmProjects\\newpg\\.venv\\Lib\\site-packages\\playwright\\_impl\\_connection.py\", line 528, in wrap_api_call\n    raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\n"}, "start": 1742284026497, "stop": 1742284032091}], "attachments": [{"name": "Exception Details", "source": "a9344a59-63be-4753-a134-73dada19cb72-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "10260902-82a0-4c3d-89c3-84455eae01f4-attachment.txt", "type": "text/plain"}], "start": 1742284025417, "stop": 1742284032100, "uuid": "1b22240f-d039-4a41-9f38-637cd0e21a04", "historyId": "f32be134b24015a44cb35606e8b44b5a", "testCaseId": "f32be134b24015a44cb35606e8b44b5a", "fullName": "testCases.errorMessageValidation_test.TestClientValidation#test_empty_payer_name_validation", "labels": [{"name": "story", "value": "Payer Name Validations"}, {"name": "severity", "value": "critical"}, {"name": "epic", "value": "Payment Gateway"}, {"name": "feature", "value": "Error Message Validations"}, {"name": "tag", "value": "critical"}, {"name": "tag", "value": "ui"}, {"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "errorMessageValidation_test"}, {"name": "subSuite", "value": "TestClientValidation"}, {"name": "host", "value": "SRS-CL-009"}, {"name": "thread", "value": "14956-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.errorMessageValidation_test"}]}